src/format-types/highlight/index.js

```diff
@@ -167,7 +169,7 @@ const HighlightButton = props => {
 			<Toolbar className="stackable-components-toolbar">
 				<Button
 					label={ __( 'Color & Highlight', i18n ) }
-					className="components-button components-icon-button components-toolbar__control"
+					className="components-button components-icon-button components-toolbar__control stackable-components-toolbar__highlight"
```

Comment:
Let's use the shorter `stk-` prefix instead `stk-components-toolbar__highlight`

```suggestion
					className="components-button components-icon-button components-toolbar__control stk-components-toolbar__highlight"
```